app/models/patient.rb

```diff
@@ -34,31 +34,36 @@ class Patient < ApplicationRecord
   }
 
   %i[address_state
-     ethnicity
      monitored_address_state
+     foreign_monitored_address_state
```

Comment:
thoughts on replacing this list with `VALID_PATIENT_ENUMS.keys`?

Reply:
That's a little tricky here, because some values from `VALID_PATIENT_ENUMS` _do_ get validated outside of the `api`/`import` context, for example see here: https://github.com/SaraAlert/SaraAlert/blob/47f64c407b23f719c346ad7a22e1a2360fb525bd/app/models/patient.rb#L23.

Reply:
But your suggestion to refactor these lists of fields out onto the ValidationHelper could work here, so that we have a centralized location indicating which fields are being validated in what way. I will see how that works out.

Reply:
got it, definitely not something that has to be done, just something we could consider in the future